svg: parse font attributes on <g> so children inherit the family

Font attributes set on a <g> element were ignored, so its children did not
inherit the family.
---
diff --git a/source/svg/svg-run.c b/source/svg/svg-run.c
index 0f825b7..3291831 100644
--- a/source/svg/svg-run.c
+++ b/source/svg/svg-run.c
@@ -1300,11 +1300,15 @@ svg_run_g(fz_context *ctx, fz_device *dev, svg_document *doc, fz_xml *root, cons
 {
 	svg_state local_state;
 	fz_xml *node;
+	/* SumatraPDF */
+	char font_family[100];
 
 	svg_begin_state(ctx, &local_state, inherit_state);
 	fz_try(ctx)
 	{
 		svg_parse_common(ctx, doc, root, &local_state);
+		/* SumatraPDF */
+		svg_parse_font_attributes(ctx, doc, root, &local_state, font_family, sizeof font_family);
 
 		for (node = fz_xml_down(root); node; node = fz_xml_next(node))
 			svg_run_element(ctx, dev, doc, node, &local_state);
